api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -234,6 +249,26 @@ public void testListClusters() throws Exception {
         .isEqualTo(testCluster);
   }
 
+  @Test
+  public void testDeleteClustersInProject() throws Exception {
```

Comment:
does not throw Exception - this can actually be removed from every method in this class